packages/components/.stylelintrc.js

```diff
@@ -7,5 +7,153 @@ module.exports = {
+
+    // while in theory this is a good idea, probably I would turn off this rule because it's buggy and triggers an error for us that doesn't make sense
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/operator-no-unspaced/README.md
+    // (many issues https://github.com/stylelint-scss/stylelint-scss/issues?q=is%3Aissue+is%3Aopen+%22operator-no-unspaced%22)
+    'scss/operator-no-unspaced': null,
+
+    // probably a good idea, but we have the make sure this will not impact consumers (may be using old versions of Sass?)
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/no-global-function-names/README.md
+    'scss/no-global-function-names': null,
+
+    // using the default one doesn't work for us (we're using BEM)
+    // found this pattern here: https://github.com/humanmade/coding-standards/pull/199
+    // see: https://stylelint.io/user-guide/rules/list/selector-class-pattern/
+    'selector-class-pattern': [
+      '^(?<block>(?:[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)(?<element>(?:__[a-z][a-z0-9]*(?:-[a-z0-9]+)*))?(?<modifier>(?:--[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)?$',
+      { resolveNestedSelectors: true },
+    ],
+    // potentially we could even use this one (I tried to use https://github.com/SunHuawei/stylelint-force-app-name-prefix/ but didn't work...)
+    // 'selector-class-pattern': [
+    //   '^hds-(?<block>(?:[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)(?<element>(?:__[a-z][a-z0-9]*(?:-[a-z0-9]+)*))?(?<modifier>(?:--[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)?$|^mock-(?:[a-z][a-z0-9]*)$',
+    //   { resolveNestedSelectors: true },
+    // ],
+
+    // the 'always-multi-line' option works for me
+    // see: https://stylelint.io/user-guide/rules/list/selector-list-comma-newline-after/
+    'selector-list-comma-newline-after': 'always-multi-line',
+
+    // strong-ish option here to disable this check: I prefer being explicit vs implicit (but if there are stronger opinions, I can live with it)
+    // see: https://stylelint.io/user-guide/rules/list/shorthand-property-no-redundant-values/
+    'shorthand-property-no-redundant-values': null,
+
+    // I would stick to the one we're using ("double") which is also the "standar" one (https://github.com/stylelint/stylelint-config-standard/blob/c12b8cb2b9a57e8dbd2696dd7ac2f78e72b829a4/index.js#L162)
+    // see: https://stylelint.io/user-guide/rules/list/string-quotes/
+    // 'string-quotes': 'double',
+    'string-quotes': null,
+
+    // since there's an issue with "currentColor" (https://github.com/stylelint/stylelint/issues/5863) I propose to simply ignore it
+    // see: https://stylelint.io/user-guide/rules/list/value-keyword-case/
+    'value-keyword-case': ['lower', { ignoreKeywords: ['currentColor'] }],
+
+    // =============================
+    // CONTROVERSIAL/HARD-TO-DECIDE:
+    // =============================
+
+    'block-closing-brace-empty-line-before': null,
+    'block-opening-brace-newline-after': null,
```

Comment:
`always` (or at least `always-multi-line` if we decide to allow single line definitions)